### DISCUSSION TITLE
Building Analyzer along with separate project

### DISCUSSION THREAD

**Discussion by LanceTheLeski** [dotnet/roslyn#66520] [Q&A]
If my VS solution has two projects, e.g. ProjectA.csproj (project in net7.0) and ProjectB.csproj (analyzer in netstandard2.0), and I wanted to build both of these into a NuGet package from ProjectA.csproj, how would I go about that? It seems like it should be simple but I cannot find a straight forward answer. I have built the analyzer on its own and it works, but I cannot reference it with anything else.

**Comment by ryzngard** [comment]
I think by convention analyzer packages have been separate, and you can add as a dependency of your package for ProjectA. That said, these instructions likely will help you get close to what you want: https://learn.microsoft.com/en-us/nuget/guides/analyzers-conventions
